src/nccl_ofi_api.cpp

```diff
@@ -575,13 +575,19 @@ ncclResult_t nccl_net_ofi_isend_v2(void* sendComm, void* data, int size,
 }
 
 
-ncclResult_t nccl_net_ofi_isend_v5(void *sComm, void* data, int size,
-				   int tag, void *mhandle, void** req)
+ncclResult_t nccl_net_ofi_isend_v5(void *sendComm, void* data, int size,
+				   int tag, void *mhandle, void** request)
+{
+	return nccl_net_ofi_isend_v9(sendComm, data, static_cast<size_t>(size), tag, mhandle, request);
```

Comment:
So you've removed the validation from _v9, which makes sense, but _v5 still has to make sure you dint overflow INT_MAX. You need it in either one of the two places.

Reply:
Never mind, what you had works for the v5 entry.